.vscode/launch.json

```diff
@@ -0,0 +1,31 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "test:vitest",
+      "type": "node",
+      "request": "launch",
+      "runtimeExecutable": "npx",
+      "runtimeArgs": ["dev-tool", "run", "test:vitest"],
+      "args": ["--", "--inspect-brk", "--no-file-parallelism", "${input:filename}"],
+      "autoAttachChildProcesses": false,
+      "skipFiles": ["<node_internals>/**"],
+      "console": "integratedTerminal",
+      "attachSimplePort": 9229,
+      "cwd": "${workspaceFolder}/sdk/${input:package-directory}"
+    }
```

Comment:
do you think it's helpful/possible to have another configuration to debug current file in the editor?

Reply:
Let me check!

Reply:
Looking at the variable reference, we may be able to do something like:

```json
   {
      "id": "filename",
      "type": "promptString",
      "description": "(Optional) Enter a part of the test file name (e.g. 'foo' will run 'foo.spec.ts', 'notfoo.spec.ts', 'test.foo.spec.ts', etc.)",
      "default": "${fileBasename}"
    }
```

Which will use the currently opened file. There are ways to be smarter about it:

1. ${relativeFile} will contain the following: `<workspaceFolder>/sdk/<serviceDirectory>/<packageDirectory>/test/public/node/myTest.spec.ts` (for example, if we have that file open)
2. There are command extensions that can be used to split and manipulate that string

We can write a launch config that takes no parameters and smartly splits out the relative path and the filename and passes those in. I suggest we do that separately and I'll create an issue to look into it 👍 It's a great idea IMO

Reply:
Created https://github.com/Azure/azure-sdk-for-js/issues/31771